src/confluent_kafka/admin/_config.py

```diff
@@ -177,3 +204,28 @@ def set_config(self, name, value, overwrite=True):
         if not overwrite and name in self.set_config_dict:
             return
         self.set_config_dict[name] = value
+
```

Comment:
I feel we should have different class for Incremental Alter configs. We are going to Remove Alter Configs in future. This implementation is confusing for now and will be difficult to reason about in future.

Reply:
Same considerations as we had in librdkafka apply here, as the ConfigResource contains a list of ConfigEntry differently from J.

Reply:
I was asking this so that we have a new constructor to have `set_incremental_config` parameter as well. We can add it here itself with default to none. Like what is happening with `set_config`.

Reply:
Done